### PR TITLE
Upgrade gitbook to 3.2.2

### PR DESCRIPTION
**Pull request by arvindsv**
Very similar to https://github.com/gocd/docs.go.cd/pull/96

Had to move all source files into a directory, since yarn didn't like directories named "1" and "2" and was looking for package.json files there.

### CODE REVIEW

**Comment by arvindsv** [comment]
If it looks ok, I'll backport.

File: source/writing_go_plugins/notification/notification_plugin_overview.md [hunk 1]
```diff
@@ -0,0 +1 @@
+!REDIRECT "https://plugin-api.gocd.org/current/notifications"
```
Comment (ketan) [inline source/writing_go_plugins/notification/notification_plugin_overview.md]: You may need to rename this in `SUMMARY.md`

**Comment by ketan** [comment]
Merge when you've fixed the SUMMARY.md file.

**Comment by arvindsv** [comment]
Ok, fixed and merged.

**Pull request merged**